Sources/CryptomatorCloudAccess/Crypto/VaultFormat6/VaultFormat6ShorteningProviderDecorator.swift

```diff
@@ -59,7 +59,7 @@ class VaultFormat6ShorteningProviderDecorator: CloudProvider {
 		}
 	}
 
-	func downloadFile(from cloudPath: CloudPath, to localURL: URL) -> Promise<Void> {
+	func downloadFile(from cloudPath: CloudPath, to localURL: URL, onTaskCreation: ((URLSessionDownloadTask?) -> Void)?) -> Promise<Void> {
```

Comment:
`onTaskCreation` has to be passed through when calling the delegate's function.

Reply:
Resolved by https://github.com/cryptomator/cloud-access-swift/pull/21/commits/0c69347b2e1810c8a9b1047ecc337c97ead30184.